function_2.py

```diff
@@ -41,7 +41,11 @@ def lambda_handler(event, context):
 
     logging.info('s3.get_all_csv()')
     files = s3.get_all_csv()
-    logging.info("Found {} csv files to copy to Redshift".format(len(files)))
+    if len(files):
+        logging.info("Found {} csv files to copy to Redshift".format(len(files)))
+    else:
+        logging.info("No csv files to copy to Redshift")
```

Comment:
Nitpick: I might add `, exiting` to the end of that log line so it's obvious from the logs that the function is intentionally exiting.

Reply:
Agree